SpeechResponder/Help.md

```diff
@@ -203,6 +203,23 @@ Common usage of this is to provide clear callsigns and idents for ships, for exa
 
    Ship ident is {ICAO(ship.ident)}.
 
+### JumpDetails()
+
+This function will provide jump information based on your ship loadout and current fuel level, dependent on the followiing types:
+
+  * `next` range of next jump at current fuel level
+  * `max` range of single jump at 'max fuel'
+  * `total` total range of multiple jumps at current fuel level
+  * `full` total range of multiple jumps at tank and reservoir capacity
```

Comment:
Not at reservoir capacity. Just tank capacity.

Reply:
Fixed.